Documentation/building/netbsd-instructions.md

```diff
@@ -6,7 +6,7 @@ This guide will walk you through building CoreCLR on NetBSD.  We'll start by sho
 Environment
 ===========
 
-These instructions are written on NetBSD 7.x on the amd64 platform, since that's the release the team uses.
+These instructions are written on NetBSD 7.x on the amd64 platform since that's the release the team uses.
```

Comment:
I prefer it with the comma; it's using the comma to represent a pause that's separating an explanation.